sw/airborne/modules/sensors/airspeed_ets.c

```diff
@@ -156,6 +156,7 @@ void airspeed_ets_read_event(void)
 
   // Continue only if a new airspeed value was received
   if (airspeed_ets_valid) {
+#ifndef AIRSPEED_ETS_3RD_PARTY_MODE      
```

Comment:
it makes more senses to check flags like this with `#if !AIRSPEED_ETS_3RD_PARTY_MODE`